eng/versioning/external_dependencies.txt

```diff
@@ -274,7 +274,7 @@ net.oneandone.reflections8:reflections8;0.11.7
 net.jonathangiles.tools:dependencyChecker-maven-plugin;1.0.6
 net.jonathangiles.tools:whitelistgenerator-maven-plugin;1.0.2
 org.apache.commons:commons-collections4;4.4
-org.apache.commons:commons-text;1.6
+org.apache.commons:commons-text;1.10.0
```

Comment:
will this impact other projects using this library?

Reply:
Only azure-cosmos and azure-cosmos-encryption use this library :)